instance/models/load_balancer.py

```diff
@@ -136,7 +137,7 @@ def get_instances(self):
             if field.one_to_many and issubclass(field.related_model, LoadBalancedInstance):
                 yield from getattr(self, field.get_accessor_name()).iterator()
 
-    def get_configuration(self):
+    def get_configuration(self, triggering_id=None):
         """
         Collect the backend maps and configuration fragments from all associated instances.
```

Comment:
@smarnach Could you document the role of the `triggering_id` in the docstrings? If I understand correctly it is compared to the instance id to let `instance.get_load_balancer_configuration()` know when the call has been triggered by a change on that instance, right? Maybe `triggering_instance_id` would be more explicit, too.